app/services/users_service.py

```diff
@@ -18,6 +19,12 @@ async def create_user(self, session: SessionDep, user_in: UserCreate) -> User:
                 telegram_id=user_dict.get("telegram_id"),
             )
             await session.refresh(user)
+            password = user_in.get_password()
+            if password is not None:
+                password = await PasswordRepository(session).create_password(
+                    user_dict.get("public_id"), password
+                )
+                await session.refresh(password)
             return user
```

Comment:
En vez de usar un doble negado, me queda más claro si directamente ponés:
- Además, por ejemplo si pongo un password vació pasa y lo agrega a la tabla porque no es None.
- En cambio con esta condición, los passwords vacíos no entran al if.
```suggestion
            if password:
                password = await PasswordRepository(session).create_password(
                    user_dict.get("public_id"), password
                )
                await session.refresh(password)
            return user
```

Reply:
Se realizo la correccion del IF